src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseBaseKernels.hpp

```diff
@@ -1457,6 +1459,7 @@ struct StatisticsKernel
           real64 & totalUncompactedPoreVol,
           arraySlice1d< real64 > const & phaseDynamicPoreVol,
           arraySlice1d< real64 > const & phaseMass,
+          arraySlice1d< real64 > const & immobilePhaseMass,
```

Comment:
```suggestion
          arraySlice1d< real64 > const & trappedPhaseMass,
```
because the `trappedPhaseMass` can be mobile.